HW5/task1.js

```diff
@@ -0,0 +1,17 @@
+let averageGrade = 80;
+
+if (averageGrade < 60) {
+    console.log ("Незадовільно");
+}
+else if (averageGrade >=60 && averageGrade <=70) {
```

Comment:
Дивись, по-факту, тут не треба це (див. виправлення нище). Бо з минулого іфа ти і так знаєш, що averageGrade>=60, просто тому що інакше воно б зайшло в перший іф
```suggestion
else if (averageGrade <=70) {
```

Всі ті самі коментарі в наступних else if та для switch case